tests/h/models/token_test.py

```diff
@@ -2,12 +2,40 @@
 
 from __future__ import unicode_literals
 
+import datetime
+
 from h.models import Token
 
 
 class TestToken(object):
-    def test_init_generates_value(self):
-        assert Token().value
+    def test_init_dev_token(self):
+        userid = 'acct:test@hypothes.is'
+
+        dev_token = Token(userid=userid)
+
+        assert dev_token.userid == userid
+        assert dev_token.value
+        assert dev_token.value.startswith("6879-")
+        assert not dev_token.expires
+        assert not dev_token.authclient
+        assert not dev_token.refresh_token
+
+    def test_init_access_token(self):
+        userid = 'acct:test@hypothes.is'
+        expires = one_hour_from_now()
+        authclient = 'example.com'
+
+        access_token = Token(userid=userid,
+                             expires=expires,
+                             authclient=authclient)
+
+        assert access_token.userid == userid
+        assert access_token.value
+        assert access_token.value.startswith("6879-")
```

Comment:
Right okay, I'm fine not to bother, just couldn't remember what happened after that discussion.